thirdeye/thirdeye-frontend/app/mirage/fixtures/applicationAnomalies.js

```diff
@@ -7,150 +7,162 @@ export default [
     start: 1491804013000,
     end: 1491890413000,
     dimensions: {
-      country: "US",
-      pageName: "some_page"
+      app: 'company',
+      country: 'US',
+      os: 'Android OS',
```

Comment:
fyi, since we're open source, we're trying to obfuscate the data as much a possible.